### DISCUSSION TITLE
Add a "Equipment Menu" section

### DISCUSSION THREAD

**Discussion by dukio** [SkyCryptWebsite/SkyCrypt#1221] [General]
With nether update coming up we could create a new top section containing the current active gear:
- Armor
- Equipment (4 new slots in nether update, they work like armor)
- Pet
- Inventory view

So the currently worn armor gets removed from wardrobe and displayed here, same with current active weapon and pet.
The inventory section is moved here.

![image](https://user-images.githubusercontent.com/2744227/159361374-d67b51db-f0cc-4402-b7ac-c52432726f5f.png)

### Quick mockup
![image](https://user-images.githubusercontent.com/2744227/158248406-420d1bb8-a82d-46e8-91db-5f9c1eb97a6a.png)

**Comment by nfurth1** [comment]
Agreed, like the wardrobe section but for equipment
